Makefile

```diff
@@ -668,7 +668,10 @@ client-binaries: examples/backendclient/backendclient test/cmd/clientloadgen/cli
 example-binaries: example-mmf-binaries example-evaluator-binaries
 example-mmf-binaries: examples/functions/golang/manual-simple/manual-simple
 example-evaluator-binaries: examples/evaluators/golang/simple/simple
-presubmit: fmt vet build test
+
+presubmit: REGISTRY = gcr.io/open-match-public-images
```

Comment:
Please add a comment explaining why these variables are being changed for presubmit.